Makefile.include

```diff
@@ -250,9 +250,22 @@ ifneq ($(GNRC_NETIF_NUMOF),1)
   CFLAGS += -DGNRC_NETIF_NUMOF=$(GNRC_NETIF_NUMOF)
 endif
 
+# list of all RIOT packages
+ALL_PKGS := $(sort $(notdir $(patsubst %/Makefile,%,$(wildcard $(RIOTPKG)/*/Makefile))))
 # process dependencies
 include $(RIOTBASE)/Makefile.dep
 
+# Detect if an application or Makefile.dep defined USEPKG
+ifneq (,$(USEPKG))
+  ifeq ($(origin USEPKG), file)
```

Comment:
I currently need this because USEPKG is exported in `vars.inc.mk` and it seems already set before executing make in murdock.

I could remove the export in `vars.inc.mk` maybe, but I still do not get why it has a value when passing through here for the moment (I did not investigate).

Reply:
There is currently a usage in `sys/shell/commands` that should be updated to use `USEMODULE`. I will try to go early in that direction.